src/lib/shutdown_manager.js

```diff
@@ -95,7 +95,7 @@ export default class ShutdownManager extends EventEmitter {
   }
 
   scheduleTerminationPoll() {
-    return async () => {
+    return (async () => {
```

Comment:
Added a bug for this because I like the improvements.  Thanks! https://bugzilla.mozilla.org/show_bug.cgi?id=1346252